spec/system/infinite_redirect_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "Infinite redirect", service: :placements, type: :system do
+  let!(:user) { create(:placements_support_user) }
+
+  scenario "When the support users permissions are removed they aren't stuck in an infinite loop" do
+    given_i_am_a_support_user
+    when_i_view_all_organisations
+    and_the_support_user_is_removed_as_a_support_user
+    and_i_click_link("Organisations")
+    then_i_no_longer_have_access_to_the_schools_details
+    and_to_be_redirected_to_the_root_path
```

Comment:
This might read more fluently:

```suggestion
    and_i_am_redirected_to_the_root_path
```